Hymns/Display/DisplayHymnViewModel.swift

```diff
@@ -66,7 +66,13 @@ class DisplayHymnViewModel: ObservableObject {
 
                     let title: Title
                     if self.identifier.hymnType == .classic {
-                        title = "Hymn \(self.identifier.hymnNumber)"
+                        var titleLyric = ""
+                        if let lyrics = hymn.lyrics.first {
+                            if let firstLine = lyrics.verseContent.first {
+                                titleLyric = ": \(firstLine)"
+                            }
+                        }
```

Comment:
I don't think you need to do this. you have access to hymn.title, and you can just use that.

So it'd be like
```
let title = ""
if hymnType == classic {
  title = "Hymn \(hymnumber): "
}
title.append(hymn.title.replacingOccurrences(of: "Hymn: ", with: ""))
```

Reply:
I think that also takes care of your punctuation problem!